mne/viz/backends/_qt.py

```diff
@@ -37,6 +37,7 @@ def _layout_initialize(self, max_width):
         pass
 
     def _layout_add_widget(self, layout, widget, stretch=0):
+        """Add a widget to an existing layout."""
```

Comment:
I would add the documentation in `_AbstractLayout` in `_abstract.py` and use `@copy_base_doc_to_subclass_doc` on `_QtLayout` and `_IpyLayout` to avoid duplication. Considering that most of the classes of the GUI API are not properly documented, let's just keep this as a suggestion for later.